### PR TITLE
Create vmss-autoscale.yml

### PR DESCRIPTION
**Pull request by yuwzho**
## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## How to Test
*  Install ansible

```bash
$ pip install ansible[azure]
```

*  [Optional] Install ansible role

```bash
$ ansible-galaxy install Azure.azure_preview_modules
$ pip install -r ~/.ansible/roles/Azure.azure_preview_modules/files/requirements-azure.txt
```

*  Get the sample playbook

```
git clone https://github.com/Azure-Samples/ansible-playbooks.git
cd ansible-playbooks
git checkout [branch-name]
```

* Test the code

```bash
ansible-playbook <filename>.yml
```

## What to Check
Verify that the playbook is successfully run.

### CODE REVIEW

File: vmss/vmss-autoscale.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# ===========
+# Use this playbook to auto scale up VMSS.
+# Use vmss-create playbook to create VMSS first
+# This sample requires Ansible 2.7
+
+---
+- hosts: localhost
+  vars:
+    resource_group: "{{ resource_group_name }}"
+    vmss_name: vmsstest
```
Comment (kyliel) [inline vmss/vmss-autoscale.yml]: How about use "myVMSS"?

File: vmss/vmss-autoscale.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Description
+# ===========
+# Use this playbook to auto scale up VMSS.
+# Use vmss-create playbook to create VMSS first
+# This sample requires Ansible 2.7
```
Comment (kyliel) [inline vmss/vmss-autoscale.yml]: This requires you already have a VMSS. If not, refer to xxx to create one.

File: vmss/vmss-autoscale.yml (hunk 1 above)
Comment (yungezz) [reply vmss/vmss-autoscale.yml]: let's extract a global variable file maybe in next pr. @zikalino

File: vmss/vmss-autoscale.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        resource_group: "{{ resource_group }}"
+        name: "{{ vmss_name }}"
+        format: curated
+      register: output_scaleset
+
+    - name: Dump scaleset info
+      debug:
+        var: output_scaleset.ansible_facts.azure_vmss[0]
+        
+    - name: create auto scaling
```
Comment (yungezz) [inline vmss/vmss-autoscale.yml]: `Create` for consistent

**Comment by Fred-sun** [comment]
@yuwzho last commit fail! Thanks!

**Comment by Fred-sun** [comment]
@yuwzho

**Comment by Fred-sun** [comment]
@yuwzho Could you help recheck this PR? How about the PR going ahead? merged or further review. I have ran the test YML and get successfully result. Thanks!

**Comment by yuwzho** [comment]
merge it, The push test fails  because the dependency (vmss_create) is not running in the sequence.

**Pull request merged**